homeassistant/components/kegtron/sensor.py

```diff
@@ -42,7 +42,6 @@
         icon="mdi:keg",
         native_unit_of_measurement=UnitOfVolume.LITERS,
         device_class=SensorDeviceClass.VOLUME,
-        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Why not use total?

Reply:
It’s just the size of the keg that is connected to the beer tap at that moment. I thought it isn’t really a total.

Reply:
`total` could still be used in such cases (as a net sensor). Long-term stats for this case, isn't as useful I would say; removing them seems to be a good fit. 👍